### PR TITLE
update skr to allow for ecdsa key and fix null reference error

### PR DESCRIPTION
**Pull request by hgarvison**
update skr to allow for ecdsa key and fix null reference error with invalid key type

### CODE REVIEW

**Review by ksayid — APPROVED** [review]

File: pkg/skr/skr.go [hunk 1]
```diff
@@ -138,7 +140,22 @@ func SecureKeyRelease(identity common.Identity, certState attest.CertState, SKRK
...
+
+		jwKey := jwk.NewECDSAPrivateKey()
+		err = jwKey.FromRaw(privateEcdsaKey)
+		if err != nil {
+			return nil, errors.Wrapf(err, "could not encode ECDSA key as JWK")
+		}
+		return jwKey, nil
 	} else {
-		return nil, errors.Wrapf(err, "released key type not supported")
+		errString := fmt.Sprintf("released key type %v not supported", kty)
```
Comment (SethHollandsworth) [inline pkg/skr/skr.go]: nit: why not Wrapf like the one above?

Comment (ksayid) [reply pkg/skr/skr.go]: I guess there isn't an `err` to wrap like the other case. maybe `errors.Errorf` could be used instead here?

Comment (SethHollandsworth) [reply pkg/skr/skr.go]: good point, I think Errorf would be a little cleaner

Comment (hgarvison) [reply pkg/skr/skr.go]: I was following the previous standard laid out in a bunch of the other files where we create new errors.

Comment (hgarvison) [reply pkg/skr/skr.go]: Also, the wrapf was causing a null reference error that Mainak's team noticed because there was no error to wrap.

**Review by ksayid — APPROVED** [review]

**Review by SethHollandsworth — APPROVED** [review]
lgtm

**Pull request merged**